engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -78,6 +63,7 @@ public void buttonStateChanged(VRControllerState_t stateBefore, VRControllerStat
      * @param pose - the controller pose - a homogenous transformation matrix.
      * @param handIndex - the hand index - 0 for left and 1 for right.
      */
+    /* commented out due to a natives issue and VR not working at the moment anyway
```

Comment:
`//TODO:` has nice discoverability in many editors, IDEs, and code analysis toosl 😉 

```suggestion
    //TODO: commented out due to a natives issue and VR not working at the moment anyway
    /*
```

Reply:
Yep not sure why I didn't do that. Obvious improvement is obvious :-) Thanks!